content/hi/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html

```diff
@@ -65,11 +65,12 @@ <h2 style="color: #3771e3;">कुबेरनेट्स पर अपना 
         <br>
 
         <div class="row">
-            <p class="col-md-8">
+            <div class="col-md-8">
+              <p>
```

Comment:
Now it looks the change helps with issue resolution. Removing the unnecessary `<p>` tag enhances the solution even further. Refer the suggestion below

```suggestion
            <div class="col-md-8">
```